### DISCUSSION TITLE
Grab (Home)tab to move SumatraPDF

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#4094] [Ideas]
Right now, one can move SumatraPDF to a different place on the desktop by dragging a very, very tine tab on the right of an existing tab.

Suggestion: 
-add an option to move SumatraPDF by dragging an -entire- tab, not just the very tiny tab, i.e. prevent from opening the file in a 2nd SumatraPDF
or, alternatively, and maybe better
-move SumatraPDF by dragging around the "Home" tab

Thanks.

![SumatraPDF suggestion-18022024 051729](https://github.com/sumatrapdfreader/sumatrapdf/assets/101186455/c264944b-b451-4c13-8477-647786d67b25)

**Comment by GitHubRulesOK** [comment]
The yellow icon was added as the Windows drag handle next to the Windows dropdown move iconِ (ALT+Space)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/34ae15de-7ee1-4d12-b8be-734020c8c7a6)

The Windows drag function is the blue handle area you show is narrow at the top (see mine is taller) and has buttons on the right so move commands are on the Left note tabs can be dragged and removed so home tab is not an option.

**Reply by giwul70** [reply]
Ah! 

**Thanks** a lot.

About the yellow icon/handle:  I did not know that it could be used to move the SumatraPDF around. It is soo much easier than that tiny tab on the right of open files.

Thanks again.
